packages/fxa-content-server/docs/pairing-architecture.md

```diff
@@ -1,358 +0,0 @@
-## Pairing Flow Architecture
-
-The goal of the pairing flow is to allow users to quickly connect a mobile browser to Firefox Sync, without having to type their password,
-Firefox Accounts allows a mobile device to connect by pairing with an authenticated Firefox profile on their PC via scanning a QR code.
-See the [Project Plan Document](https://docs.google.com/document/d/1mFf0sfEK8o1csXLeyK1x4GS9SUMIq4q8bu25LiydPew/edit#) for more information.
-
-The intended user flow operates as follows:
-
-1. On their Desktop (Authority) device, the user selects an option to connect other device, and a QR code is displayed.
-1. On their Mobile (Supplicant) device, the user selects an option to connect to Firefox Sync via pairing, which launches a QR code scanner.
-1. The user scans the QR code from Desktop with their Mobile device.
-1. The devices establish a secure communication channel, and Mobile sends an OAuth request for Desktop to authorize.
-1. After confirming on both devices, the user's Mobile is connected to their Firefox Account.
-
-The Desktop (Authority) and Mobile (Supplicant) devices will not communicate directly, because we cannot assume that they will be on
-the same network and because this is difficult to achieve from web content.
-Instead, they will exchange encrypted messages via WebSocket to a lightweight "channel server" service which can relay messages back and forth.
-The flow performs a remote authorization where the already authenticated device generates Sync keys and an OAuth code for the new device.
-PKCE facilitates the transfer of the Sync key between devices and the channel server facilitates direct communication between the two, including the final transfer of the OAuth code.
-
-![](images/pairing-flow1.png)
-
----
-
-![](images/pairing-flow2.png)
```

Comment:
nit: Do we need to delete these images as well?

Not sure if any of the current images/* files are used or if they're all orphaned:
```sh
git ls-files packages/fxa-content-server/docs/images

packages/fxa-content-server/docs/images/choose_what_to_sync.png
packages/fxa-content-server/docs/images/complete_signin.png
packages/fxa-content-server/docs/images/complete_signin_link_expired.png
packages/fxa-content-server/docs/images/confirm.png
packages/fxa-content-server/docs/images/confirm_signin.png
packages/fxa-content-server/docs/images/pairing-flow1.png
packages/fxa-content-server/docs/images/pairing-flow2.png
packages/fxa-content-server/docs/images/signin.png
packages/fxa-content-server/docs/images/verify_email.png
```

But I don't see any .png references in the current docs:
```sh
git grep -n "\.png" packages/fxa-content-server/docs/ | wc -l # 0
```

Reply:
yeah, we should delete these.  I'm not sure about the others.  My goal is to migrate all the docs in this directory so it'll get cleaned up